packages/@textlint/textlint-plugin-text/src/index.ts

```diff
@@ -1,6 +1,6 @@
 // LICENSE : MIT
 "use strict";
 import { TextProcessor } from "./TextProcessor";
-module.exports = {
+export default {
```

Comment:
I use default exports for respect to test code.
If you want to use named exports, I'll fix it.

Reply:
>  If you want to use named exports, I'll fix it.

Yes, we prefer to use named exports like https://github.com/textlint/textlint/pull/673#discussion_r410262861

Reply:
I fixed it on 31e0f92